core/src/main/scala/kafka/log/LogManager.scala

```diff
@@ -897,9 +897,7 @@ object LogManager {
     val defaultProps = KafkaServer.copyKafkaConfigToLog(config)
     val defaultLogConfig = LogConfig(defaultProps)
 
-    val topicConfigs = AdminUtils.fetchAllTopicConfigs(zkUtils).map { case (topic, configs) =>
-      topic -> LogConfig.fromProps(defaultProps, configs)
-    }
+    val (topicConfigs, failed) = zkUtils.getLogConfigs(zkUtils.getAllTopicsInCluster, defaultProps)
```

Comment:
The current behavior is to throw an exception if any ZK operation fails. We probably want to preserve this.

Reply:
@junrao  Thanks for the review
I have included IllegalArgumentException if any failed topics while fetching configs.  Updated the PR.
Not sure if IllegalArgumentException is not appropriate here.

Reply:
@omkreddy : Failed includes exception. We can just throw the one in the first failed.

Reply:
@junrao Updated the PR.